qutebrowser/browser/webkit/webkittab.py

```diff
@@ -693,6 +764,13 @@ def event_target(self):
     def shutdown(self):
         self._widget.shutdown()
 
+    def run_js_sync(self, code, *, world=None):
+        if world is not None and world != usertypes.JsWorld.jseval:
+            log.webview.warning("Ignoring world ID {}".format(world))
```

Comment:
The `world` argument only exists so that `run_js_async` has the same API with QtWebKit and QtWebEngine - so you can move this check there and remove the `world` argument (as per the comment in `browsertab.py` above)